src/flags.ts

```diff
@@ -0,0 +1,14 @@
+import { Flags } from '@oclif/core'
+
+export const env = Flags.build<[string, string]>({
+  char: 'e',
+  description: 'optional environment variables to pass to the container',
+  multiple: true,
+  parse: async input => {
+    const pair = input.split('=').map(str => str.trim())
```

Comment:
This will break if the value contains `=`, which it legitimately could (e.g. base64-encoded value with padding).

Instead, consider regex:

```
let match = input.match(/([^=]+)=(.+)/s)
if (!match) { throw new Error(...) }
return [match[1], match[2]];
```

Reply:
Ah, good catch!